### PR TITLE
[PoC] of exporting aggregated report

### PR DESCRIPTION
**Pull request by gonchik**
Hi! 

if you interested, you can use the following idea of aggregated report

Cheers,
GT

### CODE REVIEW

**Comment by 0xB10C** [comment]
Hey, what is the output in the CSV file? Why is this 'report' needed?

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ here: [`sdn_advanced.xml`][2].
 
 [1]: https://home.treasury.gov/policy-issues/financial-sanctions/specially-designated-nationals-and-blocked-persons-list-sdn-human-readable-lists
 [2]: https://www.treasury.gov/ofac/downloads/sanctions/1.0/sdn_advanced.xml
+[3]: https://ofac.treasury.gov/specially-designated-nationals-list-data-formats-data-schemas
```
Comment (0xB10C) [inline README.md:10]: Why is this reference to [3] added, but never used?

File: generate-address-list.py [hunk 2]
```diff
@@ -74,6 +86,10 @@ def write_addresses_json(addresses, asset, outpath):
     with open("{}/sanctioned_addresses_{}.json".format(outpath, asset), 'w') as out:
         out.write(json.dumps(addresses, indent=2)+"\n")
 
+def write_addresses_csv(addresses, asset, outpath):
+    with open("{}/sanctioned_addresses_{}.csv".format(outpath, asset), 'w') as out:
+        for address in addresses:
+            out.write(address+";\n")
```
Comment (0xB10C) [inline generate-address-list.py:92]: The only difference to the TXT file is the `;`. Is a CSV output really needed?

File: README.md [hunk 3]
```diff
@@ -35,6 +36,7 @@ The sanctioned addresses can be extracted with this tool from the
 [`sdn_advanced.xml`][2] file. The tools supports the following output formats:
 - `TXT` file format (one address per line)
 - `JSON` file containing a list of addresses
+- `CSV` file containing a list of addresses
```
Comment (0xB10C) [inline README.md:39]: Isn't a CSV with one column the same as a TXT file with one address per line?

File: runner.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+echo "Getting a fresh copy of the SDN list"
+
+curl -Lo sdn_advanced.xml https://www.treasury.gov/ofac/downloads/sanctions/1.0/sdn_advanced.xml
+
+echo "Generating CSV file of aggregated addresses"
+python3 generate-address-list.py -a -f CSV
+
+echo "Successfully generated CSV file of aggregated addresses"
+echo "Enjoy your day!"
```
Comment (0xB10C) [inline runner.sh:10]: I guess we'd rather do this in the CI? 

https://github.com/0xB10C/ofac-sanctioned-digital-currency-addresses/blob/main/.github/workflows/generate-lists.yml

File: generate-address-list.py (hunk 2 above)
Comment (gonchik) [reply generate-address-list.py:92]: of course, it's just  PoC. 
in the future better to put delimeter

File: README.md (hunk 3 above)
Comment (gonchik) [reply README.md:39]: usually non-IT users familiar the csv format

File: runner.sh (hunk 4 above)
Comment (gonchik) [reply runner.sh:10]: it's an example I have shown not-IT user. 
if you want I can add as an example

**Comment by 0xB10C** [comment]
I'm going to close this as I don't see the point in having a CSV file that has the exact same contents as a TXT file. If someone, for some reason, needs a CSV file: please rename the TXT to CSV.

**Pull request closed without merging**